src/transformStateWithClones.js

```diff
@@ -65,7 +65,33 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, transforms) {
-  // write code here
+  let copy = { ...state };
+  const arr = [];
+
+  for (const transform of transforms) {
+    copy = { ...copy };
```

Comment:
you've already created a copy of state earlier, no need to do it on each loop iteration